primary/Cargo.toml

```diff
@@ -31,9 +31,9 @@ tower = "0.4.12"
 crypto = { path = "../crypto" }
 network = { path = "../network" }
 types = { path = "../types" }
-
 store = { git = "https://github.com/mystenlabs/mysten-infra.git", package = "typed-store", rev = "3b7daedf91fd8937dde26e905b8114cac459b866" }
 mysten-network = { git = "https://github.com/mystenlabs/mysten-infra.git", rev = "3b7daedf91fd8937dde26e905b8114cac459b866" }
+mockall = "0.11.0"
```

Comment:
My bad, wanted to look into this and totally forgot - thanks for flagging.

So here is the thing, in order for `mockall` to work _only for testing_ , apart from including the dependency only the `dev-dependencies`, I had to mark the use of it on the `handler.rs` file only for testing as well:
```
#[cfg(test)]
use mockall::*;
```
otherwise , as expected, it would try to compile it for prod dev as well and of course the dependency would be missing. Didn't know, but learned now!